src/lib/stylesheet.js

```diff
@@ -41,7 +44,7 @@ export default class StyleSheet {
       this._rulesCount === 0,
       'optimizeForSpeed cannot be when rules have already been inserted'
     )
-    this.flush()
+    this.flush(options.flush)
```

Comment:
why do you need to pass `options.flush`?

Reply:
In case that method gets other options eventually.